### PR TITLE
Adding InfluxDB University course links to relevant areas in docs

### LINKED ISSUE #4108: Review InfluxDB courses and add to docs
Identify pages related to existing InfluxDB U courses, and add new banner shortcode.

### PR DESCRIPTION
**Pull request by peterreg**
Closes #4108

### CODE REVIEW

File: content/enterprise_influxdb/v1.9/introduction/installation/_index.md [hunk 1]
```diff
@@ -18,3 +18,5 @@ Complete the following steps to install an InfluxDB Enterprise cluster in your o
 1. [Install InfluxDB Enterprise meta nodes](/enterprise_influxdb/v1.9/introduction/installation/installation/meta_node_installation/)
 2. [Install InfluxDB data nodes](/enterprise_influxdb/v1.9/introduction/installation/installation/data_node_installation/)
 3. [Install Chronograf](/enterprise_influxdb/v1.9/introduction/installation/installation/chrono_install/)
+
+{{< influxdbu title="Installing InfluxDB Enterprise" summary="In this course you will learn about the architecture concepts for InfluxDB Enterprise and how to install InfluxDB Enterprise with step-by-step instructions." action="Take the course" link="https://university.influxdata.com/courses/installing-influxdb-enterprise-tutorial/" >}}
```
Comment (kelseiv) [inline content/enterprise_influxdb/v1.9/introduction/installation/_index.md:22]:
```suggestion
{{< influxdbu title="Installing InfluxDB Enterprise" summary="Learn about InfluxDB Enterprise architecture and how to install InfluxDB Enterprise." action="Take the course" link="https://university.influxdata.com/courses/installing-influxdb-enterprise-tutorial/" >}}
```
```suggestion
{{< influxdbu title="Installing InfluxDB Enterprise" summary="In this course you will learn about the architecture concepts for InfluxDB Enterprise and how to install InfluxDB Enterprise with step-by-step instructions." action="Take the course" link="https://university.influxdata.com/courses/installing-influxdb-enterprise-tutorial/" >}}
```

Comment (kelseiv) [inline content/enterprise_influxdb/v1.9/introduction/installation/_index.md:22]:
```suggestion
{{< influxdbu title="Installing InfluxDB Enterprise" summary="Learn about InfluxDB Enterprise architecture and how to install InfluxDB Enterprise." action="Take the course" link="https://university.influxdata.com/courses/installing-influxdb-enterprise-tutorial/" >}}
```
```suggestion
{{< influxdbu title="Installing InfluxDB Enterprise" summary="In this course you will learn about the architecture concepts for InfluxDB Enterprise and how to install InfluxDB Enterprise with step-by-step instructions." action="Take the course" link="https://university.influxdata.com/courses/installing-influxdb-enterprise-tutorial/" >}}
```

**Review by kelseiv — CHANGES_REQUESTED** [review]

**Review by kelseiv — APPROVED** [review]

File: content/enterprise_influxdb/v1.9/flux/get-started/_index.md [hunk 2]
```diff
@@ -113,3 +113,5 @@ The Functions pane provides a list of functions available in your Flux queries.
 <div class="page-nav-btns">
   <a class="btn next" href="/enterprise_influxdb/v1.9/flux/get-started/query-influxdb/">Query InfluxDB with Flux</a>
 </div>
+
+{{< influxdbu "flux-103" >}}
```
Comment (sanderson) [inline content/enterprise_influxdb/v1.9/flux/get-started/_index.md:117]: Move this up above the page nav buttons.

**Review by sanderson — APPROVED** [review]
Thanks @peterreg! These are great! Just one minor suggestion on the placement of one of the banners. Otherwise, these are gtg.

**Pull request merged**